### PR TITLE
Fix the if statement syntax

### PR DESCRIPTION
**Pull request by RobLoach**
Nits

### CODE REVIEW

**Comment by TimothyGu** [comment]
Looks much better, thanks!

File: index.js [hunk 1]
```diff
@@ -5,14 +5,10 @@ var ect = require('ect');
...
 exports.compile = function(str, options) {
-  if (!options) {
-    options = {};
-  }
-  if (!options.root) {
-    options.root = {
-      'page': str
-    };
-  }
+  options = options || {};
```
Comment (hemanth) [inline index.js:8]: @RobLoach @TimothyGu Wouldn't we be happy if we had something like `options ||= {}` or maybe even `options?=` ?sigh...

Comment (TimothyGu) [inline index.js:8]: babel playground

It doesn't really save that many space

Comment (hemanth) [inline index.js:8]: No, `babel` can't help, cause `||=` or `?=` is not a part of ES6/7!

Well we could have done

`exports.compile = (str, options = {}) =>  { }` but still we would have to check for `root`

Comment (TimothyGu) [inline index.js:8]: that's why I said babel playground: http://babeljs.io/docs/usage/playground/

Comment (hemanth) [inline index.js:8]: ah ha :ice_cream:

**Pull request merged**